clusterdock/cli.py

```diff
@@ -181,10 +185,12 @@ def main():
     if hasattr(args, 'topology'):
         # As a workaround for https://github.com/docker/for-mac/issues/2396, we write the /etc/localtime
         # into the clusterdock config directory, which can be mounted into containers.
-        if not os.path.exists(CLUSTERDOCK_CONFIG_DIRECTORY):
-            os.makedirs(CLUSTERDOCK_CONFIG_DIRECTORY)
+        clusterdock_config_directory = os.path.expanduser(args.clusterdock_config_directory)
```

Comment:
use realpath, so any symbolic links will be resolved. so this should end up as:
`clusterdock_config_directory = os.path.realpath(os.path.expanduser(args.clusterdock_config_directory))`

Reply:
Done.